akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md

```diff
@@ -423,6 +423,47 @@ akka.cluster.monitored-by-nr-of-members = 9
 `expectNoMessage()` without timeout parameter is now using a new configuration property
 `akka.test.expect-no-message-default` (short timeout) instead of `remainingOrDefault` (long timeout).
 
+### Config library resolution change
+
+The [Lightbend Config Library](https://github.com/lightbend/config) has been updated to load both `reference.conf`
+and user config files such as `application.conf` before substitution of variables used in the `reference.conf`. 
+This makes it possible to override such variables in `reference.conf` with user configuration.
+
+For example, the default config for Cluster Sharding, refers to the default config for Distributed Data, in 
+`reference.conf` like this:
+
+```ruby
```

Comment:
are we doing this by convention in docs now for config?

Reply:
it's some old pasta for config blocks, but I think we could use plain triple for those (don't know if it renders any differently)

Reply:
It gives a bit of syntax highlighting since the ruby highlighter kind of works for hocon